src/store/servers.ts

```diff
@@ -13,28 +13,37 @@ export const servers: IServer[] = [
   {
     name     : 'Piplup (Main)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242' },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it' },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
     settings : {
       Server        : { MaxPlayers: 10 },
-      Scenario      : { MergeEnabled: false },
+      Scenario      : { MergeEnabled: true },
       Shines        : { Enabled: true },
       PersistShines : { Enabled: true },
     },
   },
   {
     name     : 'Piplup (Capture Sync)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242', port: 1028 },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it', port: 1028 },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
+  },
+  {
+    name     : 'Piplup (UDP)',
+    link     : 'https://piplup.smoo.it',
+    server   : { host: 'us.smoo.raychu.xyz', port: 1029 },
+    location : { flag: 'us', name: 'US-Central' },
     settings : {
-      Server        : { MaxPlayers: 8 },
```

Comment:
The `MaxPlayers` setting should be defined here as a minimum to show when the server is offline or the JSON API doesn't work.

Reply:
i never defined it here because i'm unsure if i want to have the max be 8 or 10